### ISSUE TITLE
Location labels for Counties have extra space at beginning

### ISSUE THREAD

**Issue by wickr** [OregonDigital/OD2#2759]
### Descriptive summary

@carakey noticed some Location labels were appearing at the beginning of the facet A-Z sort display. For Locations that have the county as the base, there's an extra space included at the beginning. This is preserved in Solr and causes problems with the alpha sort. In most other places it's fine, as the space is collapsed in display. Blazegraph data looks fine. I think the string padding in the County code is acting on the beginning in this case where it isn't needed. 

```
irb(main):001:0> item = Image.find('df66pz42h')
D, [2023-02-10T15:35:29.655858 #16881] DEBUG -- :   Load LDP (98.5ms) http://fcrepo.od2-prod.svc.cluster.local.:8080/fcrepo/rest/prod/df/66/pz/42/df66pz42h Service: 523300
=> #<Image id: "df66pz42h", head: [#<ActiveTriples::Resource:0x80084 ID:<http://fcrepo.od2-prod.svc.cluster.local.:8080/fcrepo/rest/prod/df/66/pz/42/df66pz42h/list_source#g1574560>>], tail: [#<ActiveTriples::Resource:0x80098 ...
irb(main):002:0> item.location.first.fetch
=> #<Hyrax::ControlledVocabularies::Location:0x80138 ID:<https://sws.geonames.org/5713677/>>
irb(main):003:0> item.location.first.solrize
=> ["https://sws.geonames.org/5713677/", {:label=>" Benton County >> Oregon >> United States$https://sws.geonames.org/5713677/"}]
```

See `+` before Benton in URL here:
https://prod.oregondigital.org/catalog?f%5Blocation_combined_label_sim%5D%5B%5D=+Benton+County+%3E%3E+Oregon+%3E%3E+United+States&locale=en&q%5B%5D=&search_field=all_fields

If you go to Location facet, switch to A-Z sort, you'll see just counties first, before any other entries.

### Expected behavior

County location labels don't have the extra space at the beginning.

**Comment by CGillen** [comment]
Fixed by #2762 🤞

**Comment by wickr** [comment]
QA pass.
Test work made on staging: https://staging.oregondigital.org/concern/images/5h73pw12z?locale=en
Don't see an extra space in the facet links and thus don't see the sort issue in the facet viewer.
Will update production labels after deploy but looks good so far.
